environments/ccms-ebs-upgrade.json

```diff
@@ -0,0 +1,24 @@
+{
+  "account-type": "member",
+  "environments": [
+    {
+      "name": "development",
+      "access": [
+        {
+          "github_slug": "laa-ccms-migration-team",
+          "level": "sandbox",
+          "nuke": "exclude"
+        }
+      ]
+    }
+],
+"tags": {
+  "application": "ccms-ebs-upgrade",
+  "business-unit": "LAA",
+  "infrastructure-support": "laa-ccms-platform-tea-aaaaeti4e5egi736k6a4k46qtm@moj.org.slack.com",
+  "owner": "donovan.artz@digital.justice.gov.uk"
+},
+"github-oidc-team-repositories": [""],
+"go-live-date": "2023-05-26"
```

Comment:
Can we check that this is relevant? I know that's the CCMS-EBS go live date, but I don't think this is directly part of CCMS, more that it's an area for them to do R&D on upgrade activities

Reply:
It's not relevant so I will remove it

Reply:
Thank you!